### PR TITLE
[macOS] New e2e_summary benchmark fails without Cocoapods.

### PR DESCRIPTION
**Pull request by a-wallen**
`animated_complex_opacity_perf_impeller_macos__e2e_summary` and `animated_complex_opacity_perf_macos__e2e_summary` fail without Cocoapods and without entitlements in the xcconfigs.

```
[animated_complex_opacity_perf_impeller_macos__e2e_summary] [STDOUT] stdout: [  +13 ms] "flutter drive" took 2,564ms.
[animated_complex_opacity_perf_impeller_macos__e2e_summary] [STDOUT] stderr: [   +2 ms] CocoaPods not installed or not in valid state.
```

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -2429,28 +2429,30 @@ targets:
       - .ci.yaml
 
   - name: Mac animated_complex_opacity_perf_impeller_macos__e2e_summary
-    bringup: true
-    presubmit: false
+    # bringup: true
```
Comment (ricardoamador) [inline .ci.yaml:2432]: Remove commented out line.

File: .ci.yaml [hunk 2]
```diff
@@ -2429,28 +2429,30 @@ targets:
...
+          {"dependency": "gems", "version": "v3.3.14"}
         ]
       tags: >
         ["devicelab", "hostonly", "mac"]
       task_name: animated_complex_opacity_perf_impeller_macos__e2e_summary
 
   - name: Mac animated_complex_opacity_perf_macos__e2e_summary
-    bringup: true
-    presubmit: false
+    # bringup: true
```
Comment (ricardoamador) [inline .ci.yaml:2447]: Same here. you can remove this.

**Review by ricardoamador — APPROVED** [review]
LGTM

File: dev/benchmarks/macrobenchmarks/macos/Runner/Release.entitlements [hunk 3]
```diff
@@ -4,5 +4,7 @@
 <dict>
 	<key>com.apple.security.app-sandbox</key>
 	<true/>
+  <key>com.apple.security.network.client</key>
+  <true/>
```
Comment (jmagman) [inline dev/benchmarks/macrobenchmarks/macos/Runner/Release.entitlements:8]: This whitespace looks wrong, did Xcode do this?

Comment (jmagman) [reply dev/benchmarks/macrobenchmarks/macos/Runner/Release.entitlements:8]: Also what changed that made this required, why was it passing before?

**Pull request merged**